lib/mongo/crypt/handle.rb

```diff
@@ -266,6 +266,10 @@ def set_kms_providers_local(kms_providers)
       # Validate and set the aws KMS provider information on the underlying
       # mongocrypt_t object and raise an exception if the operation fails
       def set_kms_providers_aws(kms_providers)
+        unless kms_providers[:aws]
+          raise ArgumentError.new('The :aws KMS provider must not be nil.')
```

Comment:
Usually exception messages in Ruby do not have the trailing period. This allows code to chain messages knowing the periods wouldn't be doubled up unnecessarily by following this pattern: `"#{exc}. Some other text"`